### ISSUE TITLE
[BUG] Broken menu after resize

### ISSUE THREAD

**Issue by a-rose** [aristocratos/bashtop#124]
**Describe the bug**

After decreasing the window width, the menu is not displayed correctly (Options, Help and Quit lines). Options and help contents are still fine.

**To Reproduce**

- Start bashtop in a terminal window
- Resize the window to decrease width
- Press esc or m to show the menu
- The menu lines are broken.

**Expected behavior**

- The menu lines should be displayed correctly after resizing the window.

**Screenshots**

<img width="1289" alt="Screenshot 2020-05-25 at 11 40 34" src="https://user-images.githubusercontent.com/16797333/82801523-bbf94e80-9e7d-11ea-85f3-3eb2cc3d1114.png">

**Info (please complete the following information):**
 - Bashtop version: `0.9.2`
 - Linux distribution and version: `Mac OS 10.15.4`
 - Terminal used: `iTerm2`
 - Font used: `Monaco regular 12`
 - Bash version, "bash --version" (version 4.4 or above is required):  `GNU bash, version 5.0.17(1)-release (x86_64-apple-darwin19.4.0)`
 - Locales: output of "locale -v"
```LANG=""
LC_COLLATE="C"
LC_CTYPE="UTF-8"
LC_MESSAGES="C"
LC_MONETARY="C"
LC_NUMERIC="C"
LC_TIME="C"
LC_ALL=
```

---
Thanks for bringing bashtop to MacOS ! :)

**Comment by aristocratos** [comment]
This seems to be specific to iTerm2 and only when resizing the terminal to a smaller size, can't replicate on Linux, FreeBSD or in the standard terminal on OSX.

Seems to be fixed after I added a screen clear on resize, but the terminal will look messed up after exiting bashtop if you made the window smaller while running. Nothing I can do about that.

Will add the fix in next update later today.

**Comment by aristocratos** [comment]
Should be fixed in v0.9.3
Reopen this issue if problem persists.
